MdePkg/Library/StackCheckFailureHookLibNull/StackCheckFailureHookLibNull.inf

```diff
@@ -1,24 +1,20 @@
 ## @file
-#  Provides the handler when a stack cookie check failure occurs.
+#  Library provides a hook called when a stack cookie check fails.
 #
 #  Copyright (c) Microsoft Corporation. All rights reserved.
 #  SPDX-License-Identifier: BSD-2-Clause-Patent
 ##
 
 [Defines]
   INF_VERSION                    = 0x00010005
-  BASE_NAME                      = StackCheckFailureLibNull
+  BASE_NAME                      = StackCheckFailureHookLibNull
   FILE_GUID                      = 0a4f7cde-a567-485d-8c9a-98ea271ba322

```

Comment:
nit: I tend to change the file guid when renaming a file, to avoid duplicate in existing build dirs, but probably doesn't matter too  much

Reply:
Good call -- updated the guid